src/pages/tasks/TaskDescriptionPage.js

```diff
@@ -48,6 +48,17 @@ function TaskDescriptionPage(props) {
 
     const inputRef = useRef(null);
 
+    // Same as NewtaskDescriptionPage, use the selection to place the cursor correctly if there is prior text
+    const [selection, setSelection] = useState({start: 0, end: 0});
+
+    // eslint-disable-next-line rulesdir/prefer-early-return
```

Comment:
Why disabled?

Reply:
I didn't think doing an early return here made much sense. It would've been 3 additional lines of code `if (!condition) { return; }` which don't really add anything.